target/linux/realtek/dts-5.10/rtl8380_d-link_dgs-1210-10p.dts

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "rtl838x_d-link_dgs-1210_soc.dtsi"
+
+/ {
+	compatible = "d-link,dgs-1210-10p", "realtek,rtl838x-soc";
+	model = "D-Link DGS-1210-10P";
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <20>;
+
+		/* is this pin 30 on the external RTL8231 (&gpio1)? */
+		/*mode {
+			label = "reset";
+			gpios = <&gpio0 94 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};*/
+	};
+
+	gpio1: rtl8231-gpio {
+		compatible = "realtek,rtl8231-gpio";
+		#gpio-cells = <2>;
+		gpio-controller;
+		indirect-access-bus-id = <0>;
+	};
+};
+
```

Comment:
The section from line 9 to 26 should likely be replaced with #include "rtl83xx_d-link_dgs-1210_gpio.dtsi".

Reply:
No, I don't do changes on unfinished devices or devices I can't test. I only make 1:1 changes. So all the devices that has been included in the OpenWrt source earlier will be exactly the same after my commits.
Please see commit message part: 'Implementation:'.

Reply:
I agree that the initial patches should be no-ops, resulting in identical output. @flygarn12, I think @RaylynnKnight owns one of these -10p devices, so a Tested-by could be provided if you decided to write an extra follow-up patch. But then you would sort of be obliged to accommodate for everyone's requests for device updates, and that could delay the series. So I suggest we wait with these corrections until we get the details of the restructuring ironed out.

Reply:
@svanheule @RaylynnKnight This patch is only a collection of what we know and what is official. I can't correct every non complete dts that has been already merged in this patch, that should have been handled before when those devices was merged at the beginning.

Reply:
Of course you can't know all those details, and I don't expect you to correct any previous mistakes. You can keep the scope of this PR to what it is now. If people care enough about other devices, those fixes will get in at some point.

Reply:
@flygarn12 OK, I understand your purpose for this patch-set now.  I was kind of hoping that @paulfertser effort could be combined with this to provide good support to more D-Link devices.  But once your patch is merged perhaps those with the other D-Link devices will have more incentive to fix missing functionality.

Reply:
This commit has been removed from the PR.